### PR TITLE
Login Overhaul

### PR DESCRIPTION
**Pull request by rvaidun**
Unfortunately BeReal has completely overhauled the login workflow breaking befake.fr.
I copied over the fixes made in the pull request at https://github.com/notmarek/BeFake/pull/80

The fetch on line 189 does not work. Chrome network tools shows a 400 bad request response 
image

Running the following python code successfully returns an access token
```py
import requests

res = requests.post('https://warm-scrubland-06418.herokuapp.com/https://auth.bereal.team/token?grant_type=firebase',data={
            "grant_type": "firebase",
             "client_id": "android",
             "client_secret": "F5A71DA-32C7-425C-A3E3-375B4DACA406",
             "token": "{{YOUR ID TOKEN HERE}}",
},
             headers={
                "origin": "https://warm-scrubland-06418.herokuapp.com"
             }
)
print(res.text)
```

TLDR: Bug for fetch on line 189

### CODE REVIEW

**Comment by hubertoschusch** [comment]
> 

You need to send headers too in the grant_type=firebase url

**Comment by ryanstickley** [comment]
Using my token I was able to login as I was before but error "Something went wrong whilst getting posts" keeps coming up, not sure if this is something that you guys are gonna get to fixing soon

**Comment by pigeonmal** [comment]
That work fine for me

**Comment by rvaidun** [comment]
All issues with login should be fixed

**Comment by puss2610** [comment]
the issue hasnt been fixed for me 😕, on the website i can login with my token and will get the "error! something went wrong whilst getting posts" and if i try to use my phone to login i get "app not verified"

**Comment by rvaidun** [comment]
@puss2610 unfortunately the old tokens will not work anymore. What country is your phone number in? I have a theory that firebase verification codes don't work for all countries and I will need to implement vonage verification codes to handle the countries that are getting the app not verified error.

**Comment by rvaidun** [comment]
yeah bereal has added a second token. before i was able to use the bereal API with just the first token but they've made their API only accept the second one. The getting post error has been patched in the latest update (i am able to login with token on my personal account) Logging in with token is now possible only with the two new tokens.

**Comment by puss2610** [comment]
how would i go about finding my second token. i got my original one when there was a brief section on the website where you could copy your token

**Comment by rvaidun** [comment]
@puss2610 ive pushed a fix let me know if it works

**Comment by puss2610** [comment]
it worked slightly more, i got sent the one time password but when entering that i got "Error! Error: Invalid_Session_Info"

**Comment by rvaidun** [comment]
could you send me your number to rahul@befake.fr so I could test out this problem locally?

**Comment by puss2610** [comment]
of course thank you
